react-native/update-font-icon.js

```diff
@@ -0,0 +1,44 @@
+// Generate our react-native icons from the fonticon stylesheet and images
```

Comment:
this converts the css to what we need. a little fragile but hopefully it won't change that often